bin/test_runner.js

```diff
@@ -63,9 +73,9 @@ function processPath (path, testFiles, options) {
 
 function createTestFileList () {
   const cwd = process.cwd()
-  const hasUserSuppliedPaths = process.argv.length > 2
+  const hasUserSuppliedPaths = process.argv.length > 1
```

Comment:
missed this one

Reply:
update: I see this should have been taken care off by `prepareMainThreadExecution` 🤔

Reply:
Yep it should be good as is